frontend/apps/marketing/src/themes/code.org/index.ts

```diff
@@ -10,44 +10,48 @@ const theme = createTheme({
   cssVariables: true,
   palette: {
     primary: {
-      main: '#000000',
+      main: '#0093A4', // Teal
+    },
+    secondary: {
+      main: '#8C52BA', // Purple
+    },
+    text: {
+      primary: '#292F36', // Dark Gray
```

Comment:
I talked to @moshebaricdo and we decided to use HEX codes here instead of css variables. This will make it easier for marketing to update the theme colors in the future since MUI automatically generates shades based on the main colors.

Reply:
Do we have a shot at fixing the overline contrast error while we're here?

Reply:
Ooo good question… @moshebaricdo thoughts?